intrastat_product/data/intrastat_transaction.xml

```diff
@@ -19,62 +19,88 @@
     </record>
     <record id="intrastat_transaction_22" model="intrastat.transaction">
         <field name="code">22</field>
-        <field name="description">Replacement for returned goods (free of charge)</field>
+        <field
+            name="description"
+        >Replacement for returned goods (free of charge)</field>
     </record>
     <record id="intrastat_transaction_23" model="intrastat.transaction">
         <field name="code">23</field>
-        <field name="description">Replacement (e.g. under warranty) for goods not being returned</field>
+        <field
+            name="description"
+        >Replacement (e.g. under warranty) for goods not being returned</field>
     </record>
     <record id="intrastat_transaction_31" model="intrastat.transaction">
         <field name="code">31</field>
-        <field name="description">Movements to/from a warehouse (excluding call-off and consignment stock)</field>
+        <field
+            name="description"
+        >Movements to/from a warehouse (excluding call-off and consignment stock)</field>
     </record>
     <record id="intrastat_transaction_32" model="intrastat.transaction">
         <field name="code">32</field>
-        <field name="description">Supply for sale on approval or after trial (including call-off and consignment stock)</field>
+        <field
+            name="description"
+        >Supply for sale on approval or after trial (including call-off and consignment stock)</field>
```

Comment:
Why these changes in the migration commit?

Reply:
In a perfect world...